oojs.md

```diff
@@ -24,13 +36,24 @@ Create an `Album` class that inherits from `Playlist`. It should also...
 - Have an `artist` (String) property that is determined by some input
 
 ```js
-Your answer goes here
+let Album extends playlist {
+  constructor(artist, title, song) {
+    super(title, song);
+    artist = []
+
+    }
+    newArtist(){
+      return `This artist is ${this.artist}`
+    }
+  }
+
 ```
 
 ### 4
 
 Create an instance of the `Album` class.
 
 ```js
-Your answer goes here
+const album3 = new Album("Katy Perry", "Witness" "Swish Swish");
```

Comment:
Missing a comma between parameters. Also, the songs argument is supposed to be an array of song(s).